starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +23,37 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord (words){
+  for ( i = 0; i < words.length; i ++){
+    lengthNumber += words[i].length;
+    return Math.max(lengthNumber)
+  }
+}
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray (numbers){
+  var total = 0;
+  for( ix = 1; i < numbers.length - 1; ix++ ) {
```

Comment:
If you have `i < numbers.length - 1` you will go throw the loop but missing the last value of the array